guides/v2.0/coding-standards/technical-vision/webapi.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: default
+group: coding-standards
+title: WebApi technical vision
+version: 2.0
+github_link: coding-standards/technical-vision/webapi.md
+---
+
+Web API is crucial for Magento because of the need to integrate with order management, customer management and other enterprise management software systems.
+
+There are a lot of headless magento installations, where Magento functionality is used only partially, while the other pieces of eCommerce website are provided by other systems. 
+
+See [more details](https://en.wikipedia.org/wiki/Web_API) about importance of web APIs in modern web applications.
+
+
+### Components Dependencies
+Below are some basic examples of what you can do with markdown.
```

Comment:
I think this is text from a template accidentally left in

Reply:
Good point, thanks